test_regress/t/t_c_this.pl

```diff
@@ -10,18 +10,21 @@
 
 scenarios(vlt => 1);
 
-compile();
+if ($Self->{dynamic_scheduler}) {
+    skip("Test not supported with the dynamic scheduler");
+} else {
```

Comment:
It's because `globOne` would find more than one file and fail. This test for sure can be rewritten, but I'm not sure how to approach this. Iterate over all files and check them individually? Concatenate all globbed files and then check them?

Reply:
I have happend to have already fixed this one on a branch, feel free to nick it: 
https://github.com/gezalore/verilator/blob/02b7512c1f4962752ec11a50c0e19df4e5ff9cc1/test_regress/t/t_c_this.pl#L18-L32

Reply:
Thanks, used your change. Though I guess we should put it in a pre-PR as well.